app/view_models/search_result.rb

```diff
@@ -29,8 +30,8 @@ def caseworker
     submission.assignments.first&.display_name || I18n.t('search.unassigned')
   end
 
-  def date_updated
-    submission.updated_at.to_fs(:stamp)
+  def last_updated
```

Comment:
I would much prefer a solution that solves this for the 4 places that use submission.updated_at all together, so that we don't risk having the caseworker app show different dates for the same application in different places. Having the search result show one thing, and the application details page (and the open/closed/your applications table) show another, is not ideal.

That being said, now that we have a viable search endpoint, maybe the ideal thing would be for the other tables to actually be populated by the search endpoint, and frankly the application details view to be populated by it too...